Source/Core/DolphinWX/LogWindow.cpp

```diff
@@ -359,5 +361,13 @@ void CLogWindow::Log(LogTypes::LOG_LEVELS level, const char* text)
   if (msgQueue.size() >= MSGQUEUE_MAX_SIZE)
     msgQueue.pop();
 
-  msgQueue.emplace(static_cast<u8>(level), StrToWxStr(text));
+  // Convert from UTF-8
+  wxString encoded_message = StrToWxStr(text);
+  // Fallback to OS locale if conversion failed
+  if (encoded_message.empty())
+  {
+    PanicAlert(StringFromFormat("Conversion from UTF-8 failed with: \"%s\"", text).c_str());
```

Comment:
If the message would be completely in Japanese (assuming Shift-JIS), then yes, the message would be completely unreadable. But the strings also contain the source file, source line and log type in ASCII (assuming nobody creates source files with non-ASCII names :)). So even if we don't know what the message is supposed to say, we can track down the buggy code.